mocks/ctrlrt/client.go

```diff
@@ -0,0 +1,176 @@
+// Code generated by mockery v2.2.2. DO NOT EDIT.
```

Comment:
+1 for generating mocks for the `ctrlrt/client`, it looks like the fake client is far from being complete https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/client/fake/doc.go#L27-L36